ego/reference/quoteprov.md

```diff
@@ -4,9 +4,7 @@ For remote attestation, both the attester and the verifier expect a *quote provi
 ## Azure
 Install the Azure DCAP client:
 ```bash
-wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add
-sudo add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/ubuntu/`lsb_release -rs`/prod `lsb_release -cs` main"
-sudo apt install az-dcap-client
+ego install az-dcap-client
 ```
```

Comment:
````suggestion
ego install az-dcap-client
```
Or manually:
```bash
wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add
sudo add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/ubuntu/`lsb_release -rs`/prod `lsb_release -cs` main"
sudo apt install az-dcap-client
```
````
I think we should always give both ways because as Nils pointed out some people don't like letting a new tool installing things and giving it sudo

Reply:
Yes. I also add a sudo to this one

Reply:
pls do the same for troubleshoot